pgjdbc/src/test/java/org/postgresql/test/jdbc2/Jdbc2TestSuite.java

```diff
@@ -107,6 +108,7 @@
     PGTimeTest.class,
     PgSQLXMLTest.class,
     PreparedStatementTest.class,
+    PrintDebugInfoTest.class,
```

Comment:
We do not use test suites

Reply:
I don't understand. What's the issue here?

Reply:
I think we should remove `*TestSuite.java` files. They are explicitly excluded now (see `exclude("**/*Suite*")` in `build.gradle.kts`), so I would refrain from making changes to Suite files.

Reply:
If they're gone sure but as long as they're still there it makes sense to keep them in sync.

So the gradle task ignores the suites entirely and finds everything in src/test that's a JUnit test?